Sources/Swiftly/Install.swift

```diff
@@ -56,25 +56,35 @@ struct Install: SwiftlyCommand {
     ))
     var token: String?
 
+    @Flag(help: "Skip PGP verification of the installed toolchain's signature.")
+    var noVerify = false
```

Comment:
Having an argument called `noVerify` means you have double negatives all over the place. Replace it with this and you get a `--no_verify` argument, as well as the `--verify`
```swift
    @Flag(inversion: .prefixedNo, help: "Perform PGP verification of the installed toolchain's signature.")
    var noVerify = false
```

Reply:
good idea, updated to this.